NVDSuppressions.xml

```diff
@@ -10,4 +10,284 @@
+        See https://github.com/dotnet/announcements/issues/112 for full MS Details on the CVE
+          A denial of service vulnerability exists when .NET Framework or .NET Core improperly handle web requests, aka '.Net Framework and .Net Core Denial of Service Vulnerability'. This CVE ID is unique from CVE-2019-0820, CVE-2019-0981.
+          Vunerability applies to System.Private.Uri version	4.3.0 and 4.3.1. System.Private.Uri is part of the .NetCore runtime, and fixed with version System.Private.Uri version 4.3.2.
+        FSS is pinned to .Net core SDK > 3.1.408 which is not vunerable, and deployed to Azure App Service which will have the security patch applied.
+    ]]>
+		</notes>
+		<cve>CVE-2019-0980</cve>
+	</suppress>
+	<suppress>
+		<notes>
+			<![CDATA[
+      file name: Microsoft.AspNetCore.Hosting.Abstractions.dll
+        See https://github.com/dotnet/announcements/issues/113 for full MS Details on the CVE
+        Vunerability applies to System.Private.Uri version	4.3.0 and 4.3.1. System.Private.Uri is part of the .NetCore runtime, and fixed with version System.Private.Uri version 4.3.2.
+        The issue is fixed:         
+          2.1.11 or SDK 2.1.507 for Visual Studio 2017 or SDK 2.1.604 for Visual Studio 2019
+          2.2.5 or SDK 2.2.107 for Visual Studio 2017 or SDK 2.2.204 for Visual Studio 2019
+        ERP Facade is pinned to .Net core SDK > 3.1.408 which is not vunerable, and deployed to Azure App Service which will have the security patch applied.
+    ]]>
+		</notes>
+		<cve>CVE-2019-0981</cve>
+	</suppress>
+	<suppress>
+		<notes>
+			<![CDATA[
+      file name: Microsoft.AspNetCore.Hosting.Abstractions.dll
+        See https://github.com/dotnet/announcements/issues/121 for full MS Details on CVE.
+        A denial of service vulnerability exists when .NET Core improperly handles web requests. An attacker who successfully exploited this vulnerability could cause a denial of service against a .NET Core web application. The vulnerability can be exploited remotely, without authentication.
+      This CVE applies to any .NET Core based application running on .NET Core 2.1.0-2.1.12, or .NET Core 2.2.0-2.2.6. FSS is pinned to .Net core SDK > 3.1.408 which is not vunerable, and deployed to Azure App Service which will have the security patch applied.
+    ]]>
+		</notes>
+		<cve>CVE-2019-1301</cve>
+	</suppress>
+	<suppress>
+		<notes>
+			<![CDATA[
+   file name: Microsoft.AspNetCore.Hosting.Abstractions.dll
+    An information disclosure vulnerability exists in .NET Framework and .NET Core which allows bypassing Cross-origin Resource Sharing (CORS) configurations, aka ".NET Framework Information Disclosure Vulnerability."
+    This applies to version of System.Net.Http as deployed as part of .Net core below 2.1.7 or 2.2.1. ERP Facade is pinned to .Net core SDK > 3.1.408 which is not vunerable, and deployed to Azure App Service which will have the security patch applied.
```

Comment:
ERP Facade is pinned to .Net core SDK > 6.0.101

Reply:
Done, Thanks